modules/covid_vaccine/spec/services/covid_vaccine/v0/enrollment_upload_service_spec.rb

```diff
@@ -20,18 +20,23 @@
   end
 
   context 'sftp interactions' do
-    let(:host) { 'fake_host' }
-    let(:username) { 'fake_username' }
-    let(:password) { 'fake_password' }
+    let(:host) { 'mysftp_host' }
+    let(:username) { 'mysftp_username' }
+    let(:password) { 'mysftp_password' }
+    let(:port) { 9999 }
     let(:sftp_connection_double) { double(:sftp_connection_double, upload!: true, download!: true) }
     let(:sftp_double) { double(:sftp, sftp: sftp_connection_double) }
     let(:handler) { CovidVaccine::V0::EnrollmentHandler }
 
     it 'responds to send_enrollment_file' do
```

Comment:
Is `send_enrollment_file` being invoked anywhere?

Reply:
Good call, method got renamed but spec description did not